### ISSUE TITLE
ECS support

### ISSUE THREAD

**Issue by dkniffin** [dtan4/terraforming#297]
It looks like the tool doesn't support ECS currently. It would be awesome if it did :)

**Comment by zbruhnke** [comment]
Not my project but it should support ECS as ECS is just an abstraction of raw EC2 instances, security groups, launch configurations, auto scaling groups etc. An ECS cluster is simply made up of a handful of resources and grouped together.

**Comment by o6uoq** [comment]
+1

**Comment by dkniffin** [comment]
@zbruhnke ECS creates those resources, but there's more to it than that. With terraform, you can create the [cluster](https://www.terraform.io/docs/providers/aws/r/ecs_cluster.html), the [service](https://www.terraform.io/docs/providers/aws/r/ecs_service.html), and the [task definition](https://www.terraform.io/docs/providers/aws/r/ecs_task_definition.html). You also often need an [ELB](https://www.terraform.io/docs/providers/aws/r/elb.html), with [ELB attachments](https://www.terraform.io/docs/providers/aws/r/elb_attachment.html), which I can't remember if this tool supports.

**Comment by o6uoq** [comment]
Agreed, ECS support would add a lot of value to terraforming (especially for me!). Hope to see it soon.

**Comment by rafilkmp3** [comment]
+1 here

**Comment by teamfighter** [comment]
Subscribed

**Comment by joannbrereton** [comment]
+1

**Comment by jasonventresca** [comment]
+1, subscribed

**Comment by marcossv9** [comment]
+1

**Comment by waqark3389** [comment]
subscribe + thumbs up!

**Comment by lbardelli** [comment]
+ That will definitely  helpful !

**Comment by Sirclicksalot** [comment]
+1
